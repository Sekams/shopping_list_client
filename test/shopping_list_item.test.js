import React from 'react';
import ShoppingListItem from '../src/components/shopping_list_item'

test("renders the shopping_list_item component", () => {
    const wrapper = shallow(
        <ShoppingListItem />
    );
    expect(wrapper).toMatchSnapshot();
});