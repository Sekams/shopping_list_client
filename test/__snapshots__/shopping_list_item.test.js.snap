// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the shopping_list_item component 1`] = `
<div
  className="list-card"
>
  <div
    className="item-container"
  >
    <div
      className="item-text"
    >
      <div>
        <h4
          className="item-name"
        />
      </div>
      <div>
        <h6>
          SHS 
        </h6>
      </div>
    </div>
    <div
      className="item-icon-holder"
    >
      <a
        href=""
        onClick={[Function]}
        title="Edit Item"
      >
        <i
          className="item-icons fa-pencil"
        />
      </a>
      <a
        href=""
        onClick={[Function]}
        title="Delete Item"
      >
        <i
          className="item-icons fa-trash"
        />
      </a>
    </div>
  </div>
  <CheckBox />
</div>
`;
